Blish HUD/_Utils/WindowUtil.cs

```diff
@@ -130,18 +137,8 @@ private enum GW : uint {
             }
         }
 
-        internal static void SetShowInTaskbar(IntPtr winHandle, bool showInTaskbar) {
-            SetWindowParam(winHandle, showInTaskbar);
-        }
-
-        internal static void SetupOverlay(IntPtr winHandle) {
-            SetWindowLong(winHandle, GWL_STYLE, CS_HREDRAW | CS_VREDRAW);
```

Comment:
I believe this doesn't belong here. See [GWL_STYLE](https://docs.microsoft.com/en-us/windows/win32/winmsg/window-styles).